openedx/core/djangoapps/waffle_utils/__init__.py

```diff
@@ -290,19 +290,21 @@ def course_override_callback(namespaced_flag_name):
             return None
         return course_override_callback
 
-    def is_enabled(self, course_id=None):
+    def is_enabled(self, course_key=None):
         """
         Returns whether or not the flag is enabled.
 
         Arguments:
-            course_id (CourseKey): The course to check for override before
+            course_key (CourseKey): The course to check for override before
             checking waffle.
         """
         # validate arguments
-        assert issubclass(type(course_id), CourseKey), "The course_id '{}' must be a CourseKey.".format(str(course_id))
+        assert issubclass(type(course_key), CourseKey), "The course_id '{}' must be a CourseKey.".format(
```

Comment:
`course_id` => `course_key`
Note: I agree this param name makes more sense in this context.  I feel like I found something weird where a course_key elsewhere wasn't actually a CourseKey, but was a string, and it gave me a lot of confusion, but _hopefully_ I'm misremembering. :)